src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.jest.tsx

```diff
@@ -172,27 +183,20 @@ describe("InfiniteScrollNewsArticle", () => {
   })
 
   it("triggers mediator with correct args", () => {
-    mockPositronql.mockReturnValue(Promise.resolve({ articles: [] }))
     mockCookies.mockReturnValue(null)
+    sd.CURRENT_USER = null
```

Comment:
I don't see `sd` mocked above so might might want to cleanup here to prevent leakage across other tests.